python/core/qgsvectorlayer.sip

```diff
@@ -685,6 +685,11 @@ class QgsVectorLayer : QgsMapLayer, QgsExpressionContextGenerator
     virtual QString getStyleFromDatabase( const QString& styleId, QString &msgError /Out/ );
 
     /**
+     * Will delete the named style corresponding to style id provided from the database
+     */
+    virtual void deleteStyleFromDatabase( const QString& styleId, QString &msgError /Out/ );
```

Comment:
Thanks @m-kuhn and @luipir for the comments.

I've followed the `saveStyleToDatabase` signature, to make `deleteStyleFromDatabase` as similar as possible.

But I agree that it does make sense to have `deleteStyleFromDatabase` returning a bool value, to indicate the success of the operation. 

So, I added a `bool` returning value and removed the `/Out/` parameter from sip.